source/linux/How_to_Guides/FAQ/How_to_run_BIST_on_a_core.rst

```diff
@@ -0,0 +1,140 @@
+##############################
+How to trigger BIST on a core?
+##############################
+
+   Built-in Self-test (BIST) is a feature that allows self testing of the memory
```

Comment:
This is actually overkill indentation, but as this doesn't put it under any invalid nodes I'll allow it.